src/main/java/it/pagopa/ecommerce/commons/documents/v2/activation/TransactionGatewayActivationData.java

```diff
@@ -0,0 +1,8 @@
+package it.pagopa.ecommerce.commons.documents.v2.activation;
+
+/**
+ * Extensible interface to handle multiple payment gateway transaction activated
+ * data
+ */
+public interface TransactionGatewayActivationData {
```

Comment:
[design] I would make this interface `sealed`, what do you think?

Reply:
Fixed with https://github.com/pagopa/pagopa-ecommerce-commons/pull/130/commits/297bb496cd320653ba116fcb1ad78fedc267f481